<launch>
    <arg name="configfile" default="/home/vdcl/GVINS_ws/src/IC-GVINS/config/county1.yaml"/>
	<!--~/GVINS_ws/src/IC-GVINS/config/gvins.yaml-->

    <node name="ic_gvins_node" pkg="ic_gvins" type="ic_gvins_ros" output="screen">
        <param name="imu_topic" value="/rt/imu"/>
        <param name="gnss_topic" value="/rt/fix"/>
        <param name="refgnss_topic" value="/rt/fix"/>
        <param name="image_topic" value="/front_camera/image_raw"/>

        <!-- <param name="imu_topic" value="/imu/data_raw"/>
        <param name="gnss_topic" value="/gps/fix"/>
        <param name="image_topic" value="/stereo/left/image_raw"/> -->

        <param name="configfile" value="$(arg configfile)"/>
    </node>

    <node pkg="tf2_ros" type="static_transform_publisher" name="world_to_map_broadcaster"
          args="0 0 0 -1 0 0 0 map world"/>
    <node name="visualisation" pkg="rviz" type="rviz" output="log" args="-d $(find ic_gvins)/../config/visualization.rviz"/>
    
</launch>
